SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/DataBrokerOperation.swift

```diff
@@ -19,45 +19,96 @@
+    var captchaService: CaptchaServiceProtocol { get }
     var vpnBypassService: VPNBypassFeatureProvider? { get }
+
+    func createScanRunner(profileQuery: BrokerProfileQueryData,
+                          stageDurationCalculator: StageDurationCalculator,
+                          shouldRunNextStep: @escaping () -> Bool) -> BrokerProfileScanSubJobWebRunning
+
+    func createOptOutRunner(profileQuery: BrokerProfileQueryData,
+                            stageDurationCalculator: StageDurationCalculator,
+                            shouldRunNextStep: @escaping () -> Bool) -> BrokerProfileOptOutSubJobWebRunning
 }
 
 public struct DefaultDataBrokerOperationDependencies: DataBrokerOperationDependencies {
     public let database: DataBrokerProtectionRepository
-    public var config: DataBrokerExecutionConfig
-    public let runner: WebJobRunner
+    public let contentScopeProperties: ContentScopeProperties
+    public let privacyConfig: PrivacyConfigurationManaging
+    public var executionConfig: DataBrokerExecutionConfig
     public let notificationCenter: NotificationCenter
     public let pixelHandler: EventMapping<DataBrokerProtectionSharedPixels>
     public let eventsHandler: EventMapping<OperationEvent>
     public let dataBrokerProtectionSettings: DataBrokerProtectionSettings
+    public let emailService: EmailServiceProtocol
+    public let captchaService: CaptchaServiceProtocol
     public let vpnBypassService: VPNBypassFeatureProvider?
 
     public init(database: any DataBrokerProtectionRepository,
-                config: DataBrokerExecutionConfig,
-                runner: any WebJobRunner,
+                contentScopeProperties: ContentScopeProperties,
+                privacyConfig: PrivacyConfigurationManaging,
+                executionConfig: DataBrokerExecutionConfig,
                 notificationCenter: NotificationCenter,
                 pixelHandler: EventMapping<DataBrokerProtectionSharedPixels>,
                 eventsHandler: EventMapping<OperationEvent>,
                 dataBrokerProtectionSettings: DataBrokerProtectionSettings,
+                emailService: EmailServiceProtocol,
+                captchaService: CaptchaServiceProtocol,
                 vpnBypassService: VPNBypassFeatureProvider? = nil) {
         self.database = database
-        self.config = config
-        self.runner = runner
+        self.contentScopeProperties = contentScopeProperties
+        self.privacyConfig = privacyConfig
+        self.executionConfig = executionConfig
         self.notificationCenter = notificationCenter
         self.pixelHandler = pixelHandler
         self.eventsHandler = eventsHandler
         self.dataBrokerProtectionSettings = dataBrokerProtectionSettings
+        self.emailService = emailService
+        self.captchaService = captchaService
         self.vpnBypassService = vpnBypassService
     }
+
+    public func createScanRunner(profileQuery: BrokerProfileQueryData,
```

Comment:
What's the reasoning for these being on Operation?

Reply:
Ah, so this is actually on `DefaultDataBrokerOperationDependencies`, but the diff is confusing because the dependencies object lives in the operation file. So my next question is: why is the dependencies object in this file? And my preferred answer is: no clue, let's move it. So I'm going to do that!